package/genhackage

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python
+
+# SPDX-License-Identifier: GPL-2.0
+
+# Please set HACKAGE_API_TOKEN or both HACKAGE_USERNAME and HACKAGE_PASSWORD to submit package list
+# to hackage!
+
+import os
+import csv
+from io import StringIO
+from sh import pacman, expac
```

Comment:
I wonder if we'd like to document the non stdlib dependencies somehow , in this case I think `python-requests` and `python-sh`

Reply:
optdepends in the package :)?

Reply:
And also `expac`,

Reply:
Yeah that's definitely one way, but it'd be nice to have it in the PR :F